SETUP_SUBMITTER_CMF.md

```diff
@@ -54,9 +56,44 @@ runuat.bat BuildPlugin -plugin="C:\deadline\deadline-cloud-for-unreal-engine\src
 
 - Copy the “package” folder above to your Unreal installation’s Plugins folder (C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService for example)
 
-## Submitter Installer
+## Python Dependencies
+
+There are several ways to install the required Python dependencies.
+
+If you've built and installed the plugin from the release branch above, you can simply install from pip.  Either install as: (Adjust paths accordingly)
```

Comment:
Nit: I think it'd help to label or outline the 4 alternative methods to choose from in this section. Maybe adding a number to each alternative or giving each a label or header (e.g. "**Install from pip**: If you've build and installed...")

Reply:
Sure will add that